benchmarks/decoders/generate_readme_data.py

```diff
@@ -83,21 +83,24 @@ def main() -> None:
             resize_device="cuda",
         ),
     )
-    df_data.append(
-        {
-            "system_metadata": {
-                "cpu_count": os.cpu_count(),
-                "system": platform.system(),
-                "machine": platform.machine(),
-                "python_version": str(platform.python_version()),
-                "is_cuda_available": str(torch.cuda.is_available()),
-            }
-        }
-    )
+    data_for_writing = {
+        "experiments": results,
```

Comment:
nit: why not call these results?

Reply:
When reading a data file, I find it more clear to see "experiments" and "system_metadata". But in the code, I think "results" is more clear that it's the return value of running the experiments.